tests/grid/grid_transform_02.cc

```diff
@@ -83,7 +83,7 @@ int main ()
             }
 
   // then compute new point locations and output the result
-  GridTools::laplace_transform  (new_points,tria);
+  GridTools::laplace_transform  (new_points,tria, static_cast<const Function<dim, double> *>(NULL), true);
```

Comment:
I believe its because the default value for the last argument is `false`, and its set to `true` here.